reference/toaztr.v1.yaml

```diff
@@ -1326,6 +1390,116 @@ paths:
+      - $ref: '#/components/parameters/simulationId'
+    get:
+      summary: Liste les factures liées à une simulation
+      operationId: getBills
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PartnerBillResourcePaginatedListResponse'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '500':
+          $ref: '#/components/responses/500'
+      description: Liste les factures liées à une simulation
+      security:
+        - toaztr_auth:
+            - bills/read
+      parameters:
+        - $ref: '#/components/parameters/page[size]'
+        - $ref: '#/components/parameters/page[cursor]'
+        - $ref: '#/components/parameters/ordering'
+      tags:
+        - billing
+  '/v1/bills/{id}':
+    parameters:
+      - $ref: '#/components/parameters/id'
+    get:
+      operationId: getBillById
+      description: Retourne une facture par ID
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PartnerBillResourceResponse'
+          headers:
+            ETag:
+              $ref: '#/components/headers/ETag'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '500':
+          $ref: '#/components/responses/500'
+      summary: Retourne une facture par ID
+      security:
+        - toaztr_auth:
+            - bills/read
+      tags:
+        - billing
+      parameters: []
+    delete:
+      operationId: deleteBill
+      description: Supprime une facture par ID
+      parameters:
+        - $ref: '#/components/parameters/IfMatch'
+      responses:
+        '200':
+          description: OK
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '409':
+          $ref: '#/components/responses/409'
+        '500':
+          $ref: '#/components/responses/500'
+      summary: Supprime une facture par ID
+      security:
+        - toaztr_auth:
+            - bills/write
+      tags:
+        - billing
+  /v1/bills:
+    parameters: []
+    post:
+      summary: Génère une facture
+      operationId: postBill
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PartnerBillResource'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: '#/components/schemas/PartnerBillingParametersResource'
```

Comment:
shouldn't be resources here, but plain parameters

Reply:
Indeed, I fixed this.